src/sqlittaaja/config.py

```diff
@@ -71,6 +88,18 @@ def __init__(self, path: str):
             self.parse(tomllib.load(file))
 
 
+def process_word_list(value, name: str):
+    if value is None:
```

Comment:
You can also use `exercise.get("must_contain", [])` to have a default value so that this isn't needed.